src/headless/Applications.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+/*
+** Stubs for applications which use the various headless APIs to do things. 
+** As headless grows, this will need to be refactored. Reading the implementations
+** in Applications.cpp helps you understand the Headless APIs we have today.
+*/
+namespace Surge
+{
+namespace Headless
+{
+namespace Apps
+{
+
+/**
+ * simpleOscillatorToStdOut
+ *
+ * Configure a single note on a couple of oscillators and dump the result
+ * as ASCII art
+ */
+void simpleOscillatorToStdOut();
+
+/**
+ * statsFromPlayingEveryPatch
+ *
+ * Play a C Major scale on every patch; and see what happens
+ */
+void statsFromPlayingEveryPatch();
+
+} // namespace Apps
+} // namespace Headless
+} // namespace Surge
```

Comment:
Seriously, just two simple local functions without any namespaces in main.cpp would be just fine:

```
static void testSimpleTone()
{
}

static void testScaleOnAllPatches()
{
}

int main(int argc, char **argv)
{
   testSimpleTone();
   testScaleOnAllPatches();
}

```

Reply:
OK I can do that. But main is gonna get messy as heck when we have 19 flavors which is why I started splitting them out. I can go either way now of course. They are simple and tiny functions.

Reply:
This commit introduces already much new structure. Maybe the best route would be to put them right now to main and then look into this after this has been landed?

Reply:
In the scope of just commit this like the amount of complexity that makes sense.